src/EDTF.php

```diff
@@ -4,13 +4,50 @@
 
 namespace EDTF;
 
-use EDTF\Contracts\DateTimeInterface;
-
 class EDTF
 {
-    public static function from(string $data): DateTimeInterface
+    public static function from(string $data): object
```

Comment:
Like this the caller still does not know how to handle the result

Reply:
This method can returns different object like an interval, season, set, etc. Do you have idea how to improve this method?

Reply:
Yeah good question. I do not have a good solution readily at hand.

The use case to cater to here is that the caller of the parser (or this static function) should know somehow which methods it can call. One approach would be `instanceof` checks. Another is to have methods like `isInternal(): bool {}` combined with `getInterval(): Interval {}`. 

If you can't think of anything better than `instanceof` checks we could just do that for now. Would still want to return an actual interface here though, so methods common to all EDTF classes can still be called without needing to do `instanceof`. That would be in line with edtf.js, where all these types implement https://github.com/inukshuk/edtf.js/blob/master/src/interface.js.

Reply:
I think we can just use an Interface. I have added a new ```ExtDateInterface``` for this.